README.md

```diff
@@ -971,6 +972,21 @@ Jest also runs tests in parallel by default, which can be problematic with Pact
 See [this issue](https://github.com/pact-foundation/pact-js/issues/10) for background,
 and the Jest [example](https://github.com/pact-foundation/pact-js/blob/master/examples/jest/package.json#L10-L12) for a working example.
 
+### Note on Angular
+
+Angular's HttpClient filters out many headers from the response object, this may cause issues when validating a response in tests.
+
+You'll need to add the additional header `Access-Control-Expose-Headers`, this will allow specified headers to be passed to the response object. This can be done in one of two ways, using `customProviderHeaders` as described [here](#modify-requests-prior-to-verification-request-filters) or by declaring the header in your contract:
+
+```js
```

Comment:
Is this bit supposed to be on the response  object or the request? (I'm assuming response).

It might be clearer to put a bit more context around this for others, although I'll rely on your experience to suggest whether or not this will be immediately obvious to other Angular users .

Reply:
It is supposed to be the response, I've added a bit more to try and make that clear.